### ISSUE TITLE
CTD on finishing character creation.

### ISSUE THREAD

**Issue by mundivore** [CleverRaven/Cataclysm-DDA#66451]
### Describe the bug

Attempting to start the game after finishing character creation causes CTD.

### Attach save file

[Maxfield.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/11860398/Maxfield.zip)

### Steps to reproduce

Attempt to start game after finishing character creation. Receive CTD error (logs provided below).

### Expected behavior

Gameplay begins.

### Versions and configuration

- OS: Windows 10
- Game Version: 8ee94d9
- Graphics Version: Tiles
- Ingame Language: English
- Mods Loaded: 
[dda]
[no_npc_food]
[personal_portal_storms]
[no_fungal_growth]
[No_Rail_Stations]
[magiclysm]
[StatsThroughSkills]
[stats_through_kills]

### Additional context

Potentially (likely?) related to #66427. Appears more tenaciously than described there. CTDs were sporadic/intermittent (~50% of attempts) at first, became more prevalent and eventually overwhelmingly prevalent over time.

Debug/crash logs: [logs.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/11860339/logs.zip)

I have been experiencing this bug for about a day now. Bug is currently occurring with very high consistency (every time I attempt to start world).

 Things that I have tried which did not fix the bug:

- New world.
- New character.
- New world settings.
- A few alternative mod orderings.

Things I have tried that mitigated bug with limited success:
- A completely fresh install (yesterday) returned the bug's occurrence rate to sporadic. Return to consistent occurrence did not happen after any particularly noticeable event. Low sample size but change seemed gradual. Corruption of some file/data? I am a clown with very limited coding ability and so that would be pure speculation.

Other subjects potentially of note:
- While bug was occurring sporadically, no further problems occurred on reloading a successfully started game. Problems only occurred on finishing & confirming character creation.
- While bug was occurring sporadically, waiting on the completed character screen for several minutes before finishing character creation appeared to reduce chance of CTD; however this could well be confirmation bias or artifact of low sample size.

I am very fresh to CDDA and development in general, so I would not know where to start with addressing this problem personally. I will help as much as possible but work retail so cannot be available consistently. Your patience will be greatly appreciated. Good luck friends.

**Comment by eturner** [comment]
I'm getting this as well. I had one failure followed by a success with yesterdays build, and enough repeated failures today that I had to check the bug reports. The character I made yesterday still loads fine.

**Comment by AnewbisOnGithub** [comment]
I have seen this twice now. I have will try to add a screencap of the crash to this post as well as the two crash logs I have saved for each time.

![CDDA CTD](https://github.com/CleverRaven/Cataclysm-DDA/assets/137598540/fd96fa4e-4620-42ea-89bf-c603bc489bde)
[crash 1.log](https://github.com/CleverRaven/Cataclysm-DDA/files/11876884/crash.1.log)
[crash 2.log](https://github.com/CleverRaven/Cataclysm-DDA/files/11876885/crash.2.log)

**Comment by RenechCDDA** [comment]
Duplicate of #66427
